mobile-app/app/screens/AppNavigator/screens/Balances/components/Announcements.tsx

```diff
@@ -21,17 +23,55 @@ export function Announcements (): JSX.Element {
     language
   } = useLanguageContext()
   const {
-    hiddenAnnouncements,
+    // hiddenAnnouncements,
```

Comment:
You can't remove hiddenAnnouncements. This will break the other feature (ability to close announcements)